common/openapi.yml

```diff
@@ -83,6 +83,33 @@ paths:
         500:
           $ref: '#/components/responses/ServerError'
 
+  /api/pipelineruns/v1/{pipelineName}/prepare:
+    parameters:
+      - $ref: '#/components/parameters/PipelineName'
+    post:
+      summary: Prepare a new pipeline run
+      operationId: preparePipelineRun
+      tags: [ pipelineRuns ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PreparePipelineRunRequestBody'
+      responses:
+        200:
+          description: File upload SAS URLs
```

Comment:
where do these descriptions end up?  a lot of our 200s just have `Success`  so im curious lol

Reply:
they show up in the example responses in Swagger. like this: 
<img width="1025" alt="image" src="https://github.com/DataBiosphere/terra-scientific-pipelines-service/assets/20929778/a406c042-f266-4a87-9159-338613cb2962">

agree that this particular description isn't quite right 😂